### ISSUE TITLE
Material3 missing artifacts for Android

### ISSUE THREAD

**Issue by msfjarvis** [JetBrains/compose-jb#2032]
Looking at Space, it can be seen that there are Android and Desktop artifacts available for [`org.jetbrains.compose.material:material`](https://maven.pkg.jetbrains.space/public/p/compose/dev/org/jetbrains/compose/material/), but [`org.jetbrains.compose.material3:material3`](https://maven.pkg.jetbrains.space/public/p/compose/dev/org/jetbrains/compose/material3/) only has the Desktop artifact. This results in IntelliJ (I think? happy to be corrected) being unable to resolve symbols when Material 3 classes are used in the common source set. I was hoping if this can be rectified somehow.

**Comment by msfjarvis** [comment]
This PR reproduces the problem where imports from `androidx.compose.material3` do not resolve in the IDE but the code compiles when built with Gradle

https://github.com/msfjarvis/compose-lobsters/pull/316

**Comment by Thomas-Vos** [comment]
![Screenshot 2022-05-04 at 21 14 07](https://user-images.githubusercontent.com/11077217/166809091-58035434-35af-419a-a3f7-8aded8ae939f.png)

**Comment by Thomas-Vos** [comment]
The code compiles fine for both Android and JVM desktop, the issue is just that code highlighting is broken.

Found a workaround, not sure how bad it is but code highlighting is back:

Add the following code to your commonMain dependencies block where you also specified compose.material3:
```
compileOnly("org.jetbrains.compose.material3:material3-desktop:1.2.0-alpha01-dev675")
```

Not sure if it will cause any issues but seems to work fine after a quick test on Android and desktop.

**Comment by Thomas-Vos** [comment]
For me also `compose.materialIconsExtended` is affected by this issue. Appears as red in common source code.

**Comment by msfjarvis** [comment]
It's not, things seem to be in a worse state with that version. Now I can't resolve even more imports than before

![image](https://user-images.githubusercontent.com/13348378/176600868-470d6165-5cae-4145-8659-b5942b60d05a.png)

**Comment by msfjarvis** [comment]
Issue is resolved in 1.2.0-alpha01-dev741

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-2032">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
